pkgs/development/libraries/freetype/default.nix

```diff
@@ -54,6 +60,10 @@ in stdenv.mkDerivation {
     ] ++
     optional useEncumberedCode ./enable-subpixel-rendering.patch;
 
+  postPatch = lib.optionalString (!useSubpixelRendering) ''
+    substituteInPlace include/freetype/config/ftoption.h --replace '#define TT_CONFIG_OPTION_SUBPIXEL_HINTING ' '#define __DONT_USE_IT '
```

Comment:
What is the effect of defining `__DONT_USE_IT` instead of commenting out or removing the original definition?